### PR TITLE
backport: 'trivial' pr9

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: contrib/debian/examples/dash.conf [hunk 1]
```diff
@@ -63,8 +63,8 @@
 # server=1 tells Dash-Qt and dashd to accept JSON-RPC commands
 #server=0
 
-# Bind to given address to listen for JSON-RPC connections. Use [host]:port notation for IPv6.
-# This option can be specified multiple times (default: bind to all interfaces)
+# Bind to given address to listen for JSON-RPC connections.
+# Refer to the manpage or bitcoind -help for further details.
```
Comment (PastaPastaPasta) [inline contrib/debian/examples/dash.conf:67]: dashd

File: doc/rapidcheck.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# RapidCheck property-based testing for Bitcoin Core
```
Comment (PastaPastaPasta) [inline doc/rapidcheck.md]: Hmm

File: test/functional/data/wallets/high_minversion/GENERATE.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+The wallet has been created by starting Bitcoin Core with the options
```
Comment (PastaPastaPasta) [inline test/functional/data/wallets/high_minversion/GENERATE.md]: drop 16845?

File: contrib/debian/examples/dash.conf (hunk 1 above)
Comment (PastaPastaPasta) [reply contrib/debian/examples/dash.conf:67]:
```suggestion
# Refer to the manpage or dashd -help for further details.
```

File: share/setup.nsi.in [hunk 4]
```diff
@@ -100,7 +100,7 @@ Section -post SEC0001
     WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayVersion "@PACKAGE_VERSION@"
     WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" Publisher "${COMPANY}"
     WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" URLInfoAbout "${URL}"
-    WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayIcon $INSTDIR\uninstall.exe
+    WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayIcon $INSTDIR\bitcoin-qt.exe
```
Comment (UdjinM6) [inline share/setup.nsi.in:103]: 16760:
```suggestion
    WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayIcon $INSTDIR\dash-qt.exe
```

File: test/functional/README.md [hunk 5]
```diff
@@ -110,7 +110,7 @@ Basic code to support P2P connectivity to a dashd.
 Utilities for manipulating transaction scripts (originally from python-bitcoinlib)
 
 #### [test_framework/key.py](test_framework/key.py)
-Wrapper around OpenSSL EC_Key (originally from python-bitcoinlib)
+Test-only secp256k1 elliptic curve implementation
```
Comment (UdjinM6) [inline test/functional/README.md:113]: 16960: this requires 15826

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,9 +1,14 @@
+<!--
+*** Please remove the following help text before submitting: ***
+
 Pull requests without a rationale and clear improvement may be closed
 immediately.
+-->
 
+<!--
```
Comment (UdjinM6) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: 16962: missing closing `-->` at eof

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: test/functional/README.md (hunk 5 above)
Comment (PastaPastaPasta) [reply test/functional/README.md]: dropped

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (PastaPastaPasta) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: done and squashed

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**